app/views/contests/_form.html.erb

```diff
@@ -46,8 +53,16 @@
     <% else %>
       <%= handle(@contest.owner) %>
     <% end %>
-
   </div>
+  <div class="field">
+    <%= f.check_box :live_scoreboard %>
+    <%= f.label :live_scoreboard %> - Show the scoreboard during the contest. If unchecked, users can only see their own score until the end of the contest.
+  </div>
+  <div class="field">
+    <%= f.check_box :only_rank_official_contestants %>
+    <%= f.label :only_rank_official_contestants %> - Exclude unofficial contestants from ranking. An official contestant is any student enrolled in a NZ school.
+  </div>
+  <br>
   <div class="actions">
     <%= f.submit %>
   </div>
```

Comment:
This `<br>` adds space between the last checkbox and the "Create Contest" button, is it intentional? There wasn't added space before, and I don't think the added space is necessary/consistent.

Reply:
I agree, it can be removed.